core/src/main/scala/org/ensime/core/AnalyzerManager.scala

```diff
@@ -275,7 +266,7 @@ class AnalyzerManager(
         }
         def receive: Receive = {
           case FullTypeCheckCompleteEvent =>
-            analyzer ! RemoveFilesReq(filesToBeDeleted.toList.map(f => richEnsimeFile(f.file).toPath.toFile))
+            analyzer ! RemoveFilesReq(filesToBeDeleted.map(toFile))
```

Comment:
this definitely isn't correct if the `File` is a `jar`. Perhaps you want to `collect` on `RawFile` ? I don't really understand why we're sending `RemoveFilesReq`, it's a bit weird.